app/views/taxons/show.html.erb

```diff
@@ -18,23 +18,42 @@
       } %>
     </div>
     <div class="govuk-grid-column-two-thirds">
-      <% presented_taxon.sections.each do |section| %>
+      <%
+        index_section_count = presented_taxon.sections.count
+        index_section_count += 1 if presented_taxon.organisations_section
+        index_section_count += 1 if presented_taxon.show_subtopic_grid?
+      %>
+      <% presented_taxon.sections.each_with_index do |section, index| %>
         <% if section[:show_section] %>
+          <% title = t(section[:id], scope: :content_purpose_supergroup, default: section[:title]) %>
           <div id="<%= section[:id] %>" class="taxon-page__section-group">
             <div class="govuk-grid-row">
               <div class="govuk-grid-column-two-thirds">
                 <%= render "govuk_publishing_components/components/heading", {
-                    text: t(section[:id], scope: :content_purpose_supergroup, default: section[:title]),
-                    heading_level: 2,
-                    margin_bottom: 3
+                  text: title,
+                  heading_level: 2,
+                  margin_bottom: 3
                 } %>
               </div>
             </div>
-            <%= render(
-                    partial: section[:partial_template],
-                    locals: { section: section }
-                )
-            %>
+            <%= content_tag(
+              :div,
+              data: {
+                module: "ga4-link-tracker",
+                ga4_track_links_only: "",
+                ga4_set_indexes: "",
+                ga4_link: {
+                  event_name: "navigation",
+                  type: "document list",
+                  index: {
+                    index_section: index + 1,
+                    index_section_count: index_section_count
+                  },
+                  section: title
```

Comment:
Where you're passing `title` to the dataLayer, you're using a translated version of the title (line 28).

Should this be changed so that it's always getting an English version of the title?

Reply:
Good shout, have updated, thanks. Getting a merge conflict trying to fit these changes into their original commits, so have settled on a separate commit.